app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json

```diff
@@ -992,6 +992,25 @@
                 "summary": "Get artifact version",
                 "description": "Retrieves a single version of the artifact content.  Both the `artifactId` and the\nunique `version` number must be provided.  The `Content-Type` of the response depends \non the artifact type.  In most cases, this is `application/json`, but for some types \nit may be different (for example, `PROTOBUF`).\n\nThis operation can fail for the following reasons:\n\n* No artifact with this `artifactId` exists (HTTP error `404`)\n* No version with this `version` exists (HTTP error `404`)\n* A server error occurred (HTTP error `500`)\n"
             },
+            "delete": {
+                "tags": [
+                    "Versions"
+                ],
+                "responses": {
```

Comment:
What would the response look like if the feature is disabled?

Reply:
Good question.  What are the choices?  

* `401` - Authorization Failed
* `403` - Forbidden
* `405` - Method Not Allowed
* `406` - Not Acceptable
* `409` - Conflict

None of those map all that well onto "Sorry, that feature is currently disabled".

I think `403` and `405` are probably the best candidates, with `405` probably getting my vote.

Reply:
I think `405` sounds like a reasonable choice for me, but let me know your thoughts

Reply:
Implemented 405 response for the case